### PR TITLE
Nutanix Interview Process

### PR DESCRIPTION
**Pull request by reeversedev**

### CODE REVIEW

File: content/companies/05-nutanix.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Nutanix
+excerpt: Know more about Nutanix's interview process ➡️
+index: 1
```
Comment (kamleshchandnani) [inline content/companies/05-nutanix.md]: make it 5

**Comment by kamleshchandnani** [comment]
Thank you so much for you PR 🥇. It's the first PR of this project 😎 
Really Appreciate 👏

File: content/companies/05-nutanix.md (hunk 1 above)
Comment (kamleshchandnani) [reply content/companies/05-nutanix.md]: Also seems like you committed `package-lock.json` by mistake. Kindly remove that?

Comment (kamleshchandnani) [reply content/companies/05-nutanix.md]: update the [README.md](https://github.com/kamleshchandnani/awesome-interview-process#-awesome-companies-list) section as well

Comment (reeversedev) [reply content/companies/05-nutanix.md]: Oops yeah, I'll update it.

**Comment by reeversedev** [comment]
> @ReverseDev Thank you so much for you PR 🥇. It's the first PR of this project 😎
> Really Appreciate 👏

This is a very nice Project @kamleshchandnani . Will help a lot of people for sure.

**Pull request merged**